sqlite3/lib/src/impl/statement.dart

```diff
@@ -55,6 +55,18 @@ class PreparedStatementImpl implements PreparedStatement {
     ];
   }
 
+  List<String?>? get _tableNames {
+    if (_bindings.sqlite3_column_table_name == null) {
+      // unsupported
+      return null;
+    }
+    final columnCount = _bindings.sqlite3_column_count(_stmt);
+    return Iterable.generate(columnCount, (i) {
```

Comment:
Small nit: Let's just use `List.generate` directly instead of calling `toList()` in the end.